manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml

```diff
@@ -0,0 +1,42 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: XlightFTP.XlightFTP
+PackageVersion: 3.9.4.1
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://www.xlightftpd.com/download/setup64.msi
+  InstallerSha256: 4304448292D7B63CE849EF7BDE83DA160649B6D7CA9B8C905B0E7697451307E0
+  ProductCode: '{5E04D858-A8C6-3CFD-D744-3754111BC0B7}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://www.xlightftpd.com/download/setup32.msi

```

Comment:
On Win11 this installer fails a virus check? @stephengillie

![image](https://github.com/microsoft/winget-pkgs/assets/12611259/9e167907-191b-44e1-96a4-9c53518b0542)

Reply:
Do you have the Security Intelligence Version of Defender that caught this?

Reply:
![image](https://github.com/microsoft/winget-pkgs/assets/12611259/c1a3454b-f170-4f92-ada0-2a4e4d9dfb68)

Reply:
Might be a false positive, but not sure